GoMeet/src/main/webapp/create-meeting-step3.html

```diff
@@ -4,7 +4,20 @@
   <head>
     <meta charset="UTF-8" />
     <title>Create Meeting</title>
+    <script src="js/load-map.js"></script>
+    <script src="js/meeting-location-dao.js"></script>
+    <script src="js/interactive-map.js"></script>
+    <script src="js/error-messages.js"></script>
```

Comment:
nit: generally good to order things alphabetically when declaring/importing them.